spark/src/main/scala/io/delta/sharing/spark/DeltaSharingClient.scala

```diff
@@ -76,7 +77,8 @@ private[sharing] case class QueryTableRequest(
   limitHint: Option[Long],
   version: Option[Long],
   timestamp: Option[String],
-  startingVersion: Option[Long]
+  startingVersion: Option[Long],
+  jsonPredicates: Option[String] = None
```

Comment:
If we are changing all the references we don't have to provide a default param? In general default param should be avoided to make the code easier to maintain.

Reply:
I had it in initially and meant to take it out, thanks.